UI/Selenium/Steps/CheckAlertsCommandCenterSteps.cs

```diff
@@ -241,45 +238,35 @@ public void ThenParticipantHasJoinedAndProgressedToWaitingRoomWithoutCompletingS
                         ExtensionMethods.FindElementWithWait(Driver, GetReadyForTheHearingPage.NextButton, _scenarioContext, TimeSpan.FromSeconds(Config.DefaultElementWait));
                         var oldUrl = Driver.Url;
                         cameraUrl = oldUrl.Replace("introduction", "participant/waiting-room");
+                        _hearing.HearingId = Driver.Url.Split('/').LastOrDefault();
+                        _scenarioContext["Hearing"] = _hearing;
                         Driver.Navigate().GoToUrl(cameraUrl);
-                        Thread.Sleep(3000);
                         Driver.SwitchTo().Alert().Accept();
                     }
                     else
                     {
-                        TestFramework.ExtensionMethods.FindElementEnabledWithWait(Driver, ParticipantHearingListPage.ContinueButton, 180).Click();
+                        ExtensionMethods.FindElementEnabledWithWait(Driver, ParticipantHearingListPage.ContinueButton, 180).Click();
                         Driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(Config.DefaultElementWait);
                     }
-                    Thread.Sleep(5000);
+
+                    ExtensionMethods.WaitForElementVisible(Driver, ParticipantWaitingRoomPage.ChooseCameraAndMicButton);
+                    Thread.Sleep(3000);
```

Comment:
Please add a comment as to why this is **required**

Reply:
@Michael This was here and set to 5 sec, I reduced to 3. And yes it's required. I tried to remove it but could see tests failing

Reply:
OK, a comment in the code would help.